content/tokio/tutorial/streams.md

```diff
@@ -305,7 +305,7 @@ The `size_hint()` method is used the same way as it is with [iterators][iter].
 Usually, when manually implementing a `Stream`, it is done by composing futures
 and other streams. As an example, let's build off of the `Delay` future we
 implemented in [Async in depth][async]. We will convert it to a stream that
-yields `()` three times at 10 ms intervals
+yields `()` any number of times at 10 ms intervals.
```

Comment:
Considering the relevance of three to the previous part of the article, perhaps a better solution is to just include a constructor that explicitly sets `rem` to three?

Reply:
That’s a fair point. 

To bounce off your comment, I could `impl Default for Interval`, giving `rem` the value of 3 and `delay` the value of a Delay instance with `when` set to the now Instant.

Reply:
I think its sufficient to use an ordinary `new` method for the constructor. It's an example, so we don't need to go all-in on traits.